frappe/desk/page/messages/messages_main.html

```diff
@@ -8,6 +8,8 @@
     			class="form-control messages-textarea"></textarea>
     	</div>
 		<div style="padding-top: 15px;">
+			<span class="text-muted small"
+				style="margin-left: 45px;">{{ __("Ctrl + Enter to post") }}</span>
```

Comment:
Being pedantic here:

if `navigator.appVersion.indexOf("Mac")!=-1` use `Cmd` instead of `Ctrl`